README.md

```diff
@@ -20,28 +20,32 @@ Supports Metadata versions from v8, which means all of Kusama (from CC1). Older
    - types must adhere to the conventions set out by polkadot decoding
       - type definitions for Polkadot (Kusama) are taken from Polkadot.js and deserialized into Rust (extras/polkadot)
 
-
-|  Chain       | Extrinsics | Storage | Map | Double Map | Blake\_2* hashes | Twox_* Hashes | Identity Hash |
-| ------------ | ---------- | ---     | --- | ---------- | ---------------- | ------------- | ------------- |
-|  Kusama      |   ✅	    |  ✅     |  ✅ |     ✅     |        ✅        |        ✅     |               |
-|  Polkadot    |   ✅	    |  ✅     |  ✅ |     ✅     |        ✅        |        ✅     |               |
-|  Westend     |   ✅       |  ✅     |  ✅ |     ✅     |        ✅        |        ✅     |               |
-|  Kulupu      |            |         |     |            |                  |         	    |               |
-
-
 Currently Supported Metadata Versions (From Kusama CC1):
 - [x] V8
 - [x] V9
 - [x] V10
 - [x] V11
 - [x] V12
 - [x] V13
-- [ ] V14
+- [x] V14
 
 ### (Tentative) Release & Maintenence
 #### Note: Release description is in no way complete because of current & active development for legacy desub types & scale-info based types. it is purely here as a record for things that _should_ be taken into account in the future
 
-- Depending on changes in legacy desub code, bump version in Cargo.toml for `extras/`, `core/`
+- Depending on changes in legacy desub code, bump version in Cargo.toml for `desub/`, `desub-current/`, `desub-legacy/`, `desub-common/`, `desub-json-resolver/`
 - note `upgrade-blocks` present [here](https://github.com/polkadot-js/api/tree/master/packages/types-known/src/upgrades) and modify the hard-coded upgrade blocks as necessary in the desub `runtimes.rs` file.
-	- NOTE: this step may-or-may not be necessary depending on `scale-info` integration in the future.
-
+- Take note of PR's that have been merged since the last release.
+	- look over CHANGELOG. Make sure to include any PR's that were missed in the `Unreleased` section.
+	- Move changes in `Unreleased` section to a new section corresponding to the version being released, making sure to keep the `Ureleased` header.
+- make a PR with these changes
+- once PR is merged, push a tag in the form `vX.X.X` (E.G `v0.1.0`)
+```bash
+git tag v0.1.0
+git push --tags origin master
```

Comment:
To avoid pushing excess or forgotten about tags, you can always push the tags specific version up as well. With the rotated releases it might make sense so that just in case no one pushes tags up that they shouldn't. 

I believe it would be `git push origin <tagname>`

Reply:
Ahh once I read the rest of the release notes i realized you took into account if there was multiple tags?

I guess the above is a preference then :) Either way works well to me

Reply:
nah I think you're totally right, it should be git push origin <tagname>